src/main/java/com/epam/ms/util/UserServiceMediator.java

```diff
@@ -0,0 +1,50 @@
+package com.epam.ms.util;
+
+import com.epam.ms.util.dto.User;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.client.WebClient;
+
+import java.util.List;
+
+/**
+ * The class is used to send request to the user-service
+ */
+@Component
+public class UserServiceMediator {
```

Comment:
Mediator is a design pattern and it helps to **[reduce the complexity and dependencies between tightly coupled objects communicating directly with one another](https://www.baeldung.com/java-mediator-pattern)**.
In our case, we're working with microservices and from one side we can call different systems and components but we don't use term Mediator.. we give `Client` name to those classes that are responsible for communication with other microservices/systems(out of our infrastructure).
Please, call it `UserClient` or `UserServiceClient` and you can create `client` folder at the root.

Reply:
Thank you, done